source/postprocess/heat_flux_densities.cc

```diff
@@ -0,0 +1,228 @@
+             p != boundary_indicators.end(); ++p, ++index)
+          global_boundary_flux_densities[*p] = global_values[index]/global_areas[index];
+      }
+
+      // now add all of the computed heat fluxes to the statistics object
+      // and create a single string that can be output to the screen
+      std::ostringstream screen_text;
+      unsigned int index = 0;
+      for (std::map<types::boundary_id, double>::const_iterator
+           p = global_boundary_flux_densities.begin();
+           p != global_boundary_flux_densities.end(); ++p, ++index)
+        {
+          const std::string name = "Outward heat flux density for boundary with id "
+                                   + Utilities::int_to_string(p->first)
+                                   + parenthesize_if_nonempty(this->get_geometry_model()
+                                                              .translate_id_to_symbol_name (p->first))
+                                   + " (" + unit +")";
+          statistics.add_value (name, p->second);
+
+          // also make sure that the other columns filled by the this object
+          // all show up with sufficient accuracy and in scientific notation
+          statistics.set_precision (name, 8);
+          statistics.set_scientific (name, true);
+
+          // finally have something for the screen
+          screen_text.precision(4);
+          screen_text << p->second << " " << unit
+                      << (index == global_boundary_flux_densities.size()-1 ? "" : ", ");
+        }
+
+      return std::pair<std::string, std::string> ("Heat flux densities for boundaries:",
+                                                  screen_text.str());
+    }
+  }
+}
+
+
+// explicit instantiations
+namespace aspect
+{
+  namespace Postprocess
+  {
+    ASPECT_REGISTER_POSTPROCESSOR(HeatFluxDensities,
+                                  "heat flux densities",
+                                  "A postprocessor that computes some statistics about "
+                                  "the (conductive) heat flux density for each boundary "
+                                  "id. The heat flux density is computed in outward "
+                                  "direction, i.e., from the domain to the outside, using the "
+                                  "formula $\frac{1}{|\\Gamma_i|} \\int_{\\Gamma_i} -k \\nabla T \\cdot \\mathbf n$ "
+                                  "where $\\Gamma_i$ is the part of the boundary with indicator $i$, "
+                                  "$k$ is the thermal conductivity as reported by the material model, "
+                                  "$T$ is the temperature, and $\\mathbf n$ is the outward normal. "
+                                  "Note that the quantity so computed does not include any energy "
+                                  "transported across the boundary by material transport in cases "
+                                  "where $\\mathbf u \\cdot \\mathbf n \\neq 0$.")
```

Comment:
Can you add something like this here?
```
Note that the `heat flux' postprocessor computes the same quantity as the one here, but not divided by the area. 
In other words, it computes the \\textit{total} heat flux through a boundary.
```
(And maybe the same text with obvious change in the other postprocessor's file.)